docs/controls/Gantt.md

```diff
@@ -202,17 +202,18 @@ ganttInstance.loadContent(dataArray);
-| actionLegend     | array    | []                                             | Refer [Action Legend](#action-legend)                                                |
-| showActionLegend | boolean  | true                                           | Toggle to show graph legend                                                          |
-| padding          | object   | `{ top: 10, bottom: 5, left: 100, right: 50 }` | Refer [Padding](Padding.md)                                                          |
-| clickPassThrough | object   | null                                           | Refer [Pass Through](#Pass-Through)                                                  |
-| pan              | object   | {}                                             | Refer [Panning](./Panning.md)                                                        |
+| Property Name    | Expected | Default                                         | Description                                                                          |
+| ---------------- | -------- | ----------------------------------------------- | ------------------------------------------------------------------------------------ |
+| bindLegendTo     | string   | null                                            | If DOM id provided, binds legend into that container (Example: `"#legendContainer"`) |
+| locale           | object   | LOCALE.en_US                                    | Locale object for X-Axis tick values                                                 |
+| throttle         | number   | (1000/60) => time in ms                         | Delay between resize of a browser window                                             |
+| dateline         | array    | []                                              | Refer [Dateline](../helpers/Dateline.md)                                             |
+| actionLegend     | array    | []                                              | Refer [Action Legend](#action-legend)                                                |
+| showActionLegend | boolean  | true                                            | Toggle to show graph legend                                                          |
+| padding          | object   | `{ top: 10, bottom: 10, left: 100, right: 50 }` | Refer [Padding](Padding.md)                                                          |
+| clickPassThrough | object   | null                                            | Refer [Pass Through](#Pass-Through)                                                  |
+| pan              | object   | {}                                              | Refer [Panning](./Panning.md)                                                        |
+| legendPadding    | object   | `{ top: 4, bottom: 4, left: 8, right: 8 }`      | Sets the legend padding and removes the carbon default margin of 8px around legend   |
```

Comment:
Why did we only update `legendPadding` property to `gantt` and `timeline` graph types? why not other graph types?

Reply:
Why did not we add `removeContainerPadding` property here and other graph types? Please correct me if I'm wrong

Reply:
I was under the impression that the new props have been added to graph.md and all graph types will use the same root structure. Updated the docs [here](https://github.com/JananiGunasekaran/carbon-graphs/commit/ff0183adf8c3d412ff99a9d2bc3eedcd1d3509d4). Other than gantt, timeline and pie, the other graph types refer to the root structure defined in Graph.md.

Reply:
> Why did not we add `removeContainerPadding` property here and other graph types? Please correct me if I'm wrong

I was under the impression that the new props have been added to graph.md and all graph types will use the same root structure. Updated the docs [here](https://github.com/JananiGunasekaran/carbon-graphs/commit/ff0183adf8c3d412ff99a9d2bc3eedcd1d3509d4). Other than gantt, timeline and pie, the other graph types refer to the root structure defined in Graph.md.

Reply:
> Why did we only update `legendPadding` property to `gantt` and `timeline` graph types? why not other graph types?

Missed to add the property for pie graph too. Updated the docs [here](https://github.com/JananiGunasekaran/carbon-graphs/commit/ff0183adf8c3d412ff99a9d2bc3eedcd1d3509d4). Other than gantt, timeline and pie, the other graph types refer to the root structure defined in Graph.md.